src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift

```diff
@@ -8,28 +8,44 @@ import HealthCertificateToolkit
 extension Name {
 
 	var fullName: String {
-		var givenName = self.givenName
-		var familyName = self.familyName
-
-		if givenName?.trimmingCharacters(in: .whitespacesAndNewlines) == "" {
-			givenName = standardizedGivenName
+		
+		let givenName: String?
+		let familyName: String
+		
+		if let value = self.givenName?.trimmingCharacters(in: .whitespacesAndNewlines), !value.isEmpty {
+			givenName = value
+		} else {
+			givenName = readableStandardizedGivenName
 		}
-
-		if familyName?.trimmingCharacters(in: .whitespacesAndNewlines) == "" {
-			familyName = standardizedFamilyName
+		
+		if let value = self.familyName?.trimmingCharacters(in: .whitespacesAndNewlines), !value.isEmpty {
+			familyName = value
+		} else {
+			familyName = readableStandardizedFamilyName
 		}
+		
+		let formatter = PersonNameComponentsFormatter()
+		formatter.style = .long
+
+		var nameComponents = PersonNameComponents()
+		nameComponents.givenName = givenName
+		nameComponents.familyName = familyName
 
-		return [givenName, familyName]
-			.compactMap { $0 }
-			.filter { $0.trimmingCharacters(in: .whitespacesAndNewlines) != "" }
-			.joined(separator: " ")
+		return formatter.string(from: nameComponents)
 	}
 
 	var standardizedName: String {
 		[standardizedGivenName, standardizedFamilyName]
-			.compactMap { $0 }
-			.filter { $0.trimmingCharacters(in: .whitespacesAndNewlines) != "" }
-			.joined(separator: " ")
+					.compactMap { $0 }
+					.filter { $0.trimmingCharacters(in: .whitespacesAndNewlines) != "" }
+					.joined(separator: " ")
 	}
 
+	private var readableStandardizedGivenName: String? {
+		return standardizedGivenName?.components(separatedBy: "<").joined(separator: " ")
```

Comment:
if we fall back to the standardized name, we should take it as it is and not replace `<`. There are more adjustments in a standardized name, and with `<`, it's pretty obvious that it's a transliterated name.